tests/illumina/main.nf.test

```diff
@@ -39,7 +41,48 @@ nextflow_pipeline {
         }
     }
 
+        test("Stub paired-end bwa") {
+        tag "bwa"
+        options "-stub-run"
+
+        when {
+            params {
+                input_type     = "illumina"
```

Comment:
Weird indentation here and other places in this file (extra whitespace before the = but not for aligning with following lines). Not sure if that's on purpose or not.

Reply:
Don't knwo what happened here - fixed it